src/spryker/harness.yml

```
@@ -86,6 +86,11 @@ attributes:
   elasticsearch:
     image: elasticsearch
     tag: 6.8.12
+  hostname_aliases: "= replace(flatten([
+      @('glue.external_hosts'),
+      @('yves.external_hosts'),
+      @('zed.external_hosts')
+    ]), @('hostname'), @('namespace'))"
```

what is the value of namespace here? before we weren't doing any replacement in nginx.yml.twig

---

https://github.com/my127/workspace/blob/0.1.x/docs/types/workspace.md says a DNS-friendly version of `@('workspace.name')`, which is from the workspace.yml line 1. It's used for the docker-compose project name.

For example `ci-spryker-sample-dynamic`.

`@('hostname')` is already `@('workspace.name') ~ '.' ~ @('domain')`, so `ci-spryker-sample-dynamic.my127.site`

This is removing the domain part - `.my127.site`

This might be the same result and be clearer:
```suggestion
    ]), '.' ~ @('domain'), '')"
```

---

Though, it makes it less obvious when comparing with:
```
  yves:
    external_hosts:
      DE: = 'yves-de-' ~ @('hostname')
```

---

It's ok, I was just not sure why we need that now 👍

---

I like the suggestion more I think.

---

Applied it after testing earlier :)